crates/basilica-sdk-python/examples/start_secure_cloud_gpu_rental.py

```
@@ -0,0 +1,305 @@
+    private_key_path = None
+    if ssh_key and ssh_key.public_key:
+        private_key_path = find_private_key_for_public_key(ssh_key.public_key)
+
+    # Convert SSH command to credentials format for use with ssh_utils
+    credentials = ssh_command_to_credentials(
+        ready_rental.ssh_command,
+        fallback_host=ready_rental.ip_address
+    )
+
+    # Display SSH connection info
+    print(f"\n  SSH Connection:")
```

Ruff (0.14.13)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)